english/references/references.md

```diff
@@ -0,0 +1 @@
+You calmly explain to the Chief Marshmallow that you come from the [Land of Ice and Snow](../../norwegian/).
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.